src/Assignments/ProblemIframe.tsx

```diff
@@ -107,6 +134,7 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
                 if (clickedButton.name === 'submitAnswers'){
                     setProblemStudentGrade(result.data.data.studentGrade);
                     setLastSubmittedAt?.(moment());
+                    updateSubmitActive(true);
```

Comment:
Shouldn't this be false?
Once submit is clicked it should be disabled?

Reply:
isClean = true, so disabled, yes. Sorry about the nomenclature, I was trying to do everything based on isClean - but now it's just a parameter that is passed to updateSubmitActive -- think of it as updateSubmitActive(isClean = true)

Reply:
Alright let's rename

Reply:
You're mistake was not using the dirty bit!